lib/api/core/models/repositories/profileRepository.js

```diff
@@ -234,7 +234,6 @@ class ProfileRepository extends Repository {
         ) {
           errorsManager.throw('missing_anonymous_role');
         }
-        this.profiles[profile._id] = profile;
```

Comment:
Why removing the RAM cache ?

Reply:
I didnt removed it, look the entire file